.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Ask a question
+    url: https://community.icinga.com/c/icinga-web-2/7
```

Comment:
What does the `7` stand for? Is this something that can change easily and require an update here from time to time? If so, it is possible to use a more permanent link here?

Reply:
no, that's just the ID the category was assigned by discourse and is the permanent link :)